### PR TITLE
Update Outdated GCP getting started guide[skip ci]

### LINKED ISSUE #5186: [DOC] Outdated GCP getting started guide
GCP getting started guide has a sample script for setting up the image that refers to quite old versions of RAPIDS and the RAPIDS Accelerator and support for Spark 3.0.  See https://github.com/NVIDIA/spark-rapids/blob/branch-22.06/docs/get-started/gpu_dataproc_packages_ubuntu_sample.sh#L129-L135.  The config suggestions in this file probably need to be revisited as well.

### PR DESCRIPTION
**Pull request by nvliyuan**
the plugin version is out of date in the GCP getting started guide,
update the plugin&xgboost jars versions and remove some configs
fix #5186

### CODE REVIEW

**Review by abellina — DISMISSED** [review]
thanks for fixing this @nartal1

**Comment by abellina** [comment]
build

File: docs/get-started/gpu_dataproc_packages_ubuntu_sample.sh [hunk 1]
```diff
@@ -126,12 +126,11 @@ function configure_gpu_isolation() {
   chmod a+rwx -R /sys/fs/cgroup/devices
```
Comment (jlowe) [inline docs/get-started/gpu_dataproc_packages_ubuntu_sample.sh:126]: 2022 copyrights

**Review by jlowe — APPROVED** [review]

**Comment by nvliyuan** [comment]
build

**Pull request merged**